core/src/main/scala/hydra/core/monitor/HydraMetrics.scala

```diff
@@ -70,7 +74,9 @@ object HydraMetrics {
     getOrCreateHistogram(lookupKey, metricName, tags).map(_.record(value))
   }
 
-  def addPromHttpMetric(topic: String, responseCode: String, path: String)(implicit ec: ExecutionContext): Unit = {
+  def addHttpMetric(topic: String, responseCode: String, path: String,
+                    startTime: Instant, partition: Option[Int] = None, offset: Option[Long] = None,
```

Comment:
Will you explain the value of `startTime` + `Latency` and what those two items tell us?

Reply:
startTime gives us when that request hit our endpoint. Latency will tell us if there are any given topics that are taking longer than we would expect. We can narrow down users with issues and see if we can help in any way.